DevCycle/DevCycleClient.swift

```diff
@@ -451,6 +443,8 @@ public class DevCycleClient {
                     }
                 }
             } catch {
+                Log.error("Error calling identifyUser for user_id \(String(describing: user.userId))", tags: ["identify"])
```

Comment:
@jsalaber added this as well, logically I also want to use a defaulted config in the event there is an error, however would this be a duplication since its already called within the inner `identifyUser`?